### ISSUE TITLE
`rasterio` dependency is missing in requirements.txt file

### ISSUE THREAD

**Issue by krishnaglodha** [geopython/geopython-workshop#136]
I am trying to run jupyter notebook locally, but in `geopython-workshop/workshop/jupyter/requirements.txt` `rasterio` is missing

**Comment by justb4** [comment]
I see, but I am hesitant to change `geopython-workshop/workshop/jupyter/requirements.txt` at this point in time.
This file is also used in the `Dockerfile` and -build where `rasterio` is installed as a Debian (`python3-rasterio`) package.  There could be potential dependency conflicts, resulting in a failing Notebook. Everything has to be tested again, begin to end. We have a stable situation now. Maybe `rasterio` was even taken out of `requirements.txt` for that reason. That file is tuned for the Ubuntu env and .deb installs in the Dockerfile.

At the workshop we will use Docker.  Custom install is at own risk. Will need its own `requirements.txt` , install guide, and base deps like GDAL and Proj. Maybe Conda Forge works better for that case. That would be a useful addition.

**Comment by krishnaglodha** [comment]
@justb4 got it! Thanks :D

**Comment by ldesousa** [comment]
I agree it is a bit too late to try this kind of change. The notebook in Section 5 is working fine as it is. 

We can give it a try later during the code sprint.
